### ISSUE TITLE
Could not find the generated ValidationList class: Annotation processor may not have been executed.

### ISSUE THREAD

**Issue by KamilLelonek** [realm/realm-java#644]
I get the following stacktrace:

```
12-05 13:02:33.310  11237-11237/com.myapplication.myproject E/AndroidRuntime﹕ FATAL EXCEPTION: main
    Process: com.myapplication.myproject, PID: 11237
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.myapplication.myproject/com.myapplication.myproject.main.MainActivity}: io.realm.exceptions.RealmException: Could not find the generated ValidationList class: Annotation processor may not have been executed.
            at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2298)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2360)
            at android.app.ActivityThread.access$800(ActivityThread.java:144)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1278)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5221)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:899)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:694)
     Caused by: io.realm.exceptions.RealmException: Could not find the generated ValidationList class: Annotation processor may not have been executed.
            at io.realm.Realm.createAndValidate(Realm.java:482)
            at io.realm.Realm.create(Realm.java:452)
            at io.realm.Realm.getInstance(Realm.java:403)
            at io.realm.Realm.getInstance(Realm.java:253)
            at com.myapplication.myproject.main.MainActivity.onCreate(MainActivity.scala:29)
            at android.app.Activity.performCreate(Activity.java:5933)
            at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1105)
            at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2251)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2360)
            at android.app.ActivityThread.access$800(ActivityThread.java:144)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1278)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5221)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:899)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:694)
```

Realm version: `compile 'io.realm:realm-android:0.75.1'`
Java `8`
Android Studio `1.0.0-rc4`

**Comment by aafa** [comment]
I'm having similar problem running realm with SBT + android-plugin.  

**What I have**
1. Annotated scala class that extends RealmObject
2. javacOptions ++= Seq("-processor", "io.realm.processor.RealmProcessor", "-proc:only")  That I added just in case
3. compileOrder := CompileOrder.JavaThenScala  

**What I see**
1. META-INF/services/javax.annotation.processing.Processor 
2. But at runtime I got
```
java.lang.IllegalArgumentException: Class is not part of the schema for this Realm
```

**Important** - the problem is only with scala classes, with java classes it works fine

I'm not very familiar with annotation processing, what should I look for to get this working?
